pkgs/by-name/cc/cc1541/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, asciidoc
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
`finalAttrs` pattern maybe?

Reply:
That's not necessary, at least now. I'd like to see it being a standard after the derivation fixed point is done.

Reply:
Resolved anyway